doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -198,7 +198,7 @@ IPC radio firmware
 Matter bridge
 -------------
 
-|no_changes_yet_note|
+* The application now stores a portion of the application code related to the nRF70 Series' Wi-Fi firmware in the external flash memory by default.
```

Comment:
Could we mention directly that because of that partitions layout changed and link to the migration guide in case customer would like to perform DFU between NCS versions?